### ISSUE TITLE
When I try to use 5 in font size, the font doesn't become smaller. This worked before

### ISSUE THREAD

**Issue by CyraxSputnik** [QuestPDF/QuestPDF#309]
**Describe the bug**
I have an invoice that needs a small font size. I set the default font size to 7 for content, for some headers is 10, but cannot be more because would use more than one page. In some parts I really need the font to be small. When I try to use 5 in font size it doesn't become smaller.

**To Reproduce**
I use code like this
```
td.Cell().ColumnSpan(13).PaddingTop(10f).PaddingLeft(3f).DefaultTextStyle(TextStyle.Default.FontSize(5).WrapAnywhere()).Column(cd =>
{
    cd.Item().Text(td =>
    {
... more code
```

**Expected behavior**
The previous version showed a really small font (good)

**Screenshots**
**Before:**
![before](https://user-images.githubusercontent.com/43629243/184689236-218bc73a-39a2-4556-a8be-f88ab86bf3a0.png)

**After the new update (2022-08-15)**
![after](https://user-images.githubusercontent.com/43629243/184689345-b23cc2b8-cd90-416f-9f9c-8bb06155e4a1.png)

**Additional context**
I also notice that doesn't work with the new default font (lato) or Segoe UI (the font used here)

**Comment by MarcinZiabek** [comment]
I suppose that every font has a slightly different definition of size. Meaning, the same font size value may produce various visual sizes depending on the font. Please try to readjust font sizes in your code. 

I understand the inconvinience of this process. However, I think that the default font change is quite important, and therefore I marked it as a breaking change.

If needed, you can always come back to the previous default font. It was Calibri.

**Comment by CyraxSputnik** [comment]
Hi, I'm sorry if I couldn't explain it correctly, that's not exactly what I meant, the problem is that the font size is not applied correctly when I define it by default in some parts of the code.

Well, in the end I had to change the font size manually on each span.

Now that I think about it more closely... I think only spans do this, because this is pretty much the only place I'm using them.

**Comment by CyraxSputnik** [comment]
![It works](https://user-images.githubusercontent.com/43629243/185035037-afbf841a-b63f-4801-8726-4f4205a5c914.png)

Everything else is spectacular! Thank you very much for the great work!

**Comment by MarcinZiabek** [comment]
Are you using the `.DefaultTextStyle()` approach somewhere? 

Recently, another user has found bug in this functionality. We optimized the algorithm a little bit too much. I am hoping that this will be fixed very soon, possibly today 😁

**Comment by CyraxSputnik** [comment]
Ah yes! This is exactly what happened

I have it here, at the beginning

```
container.Page(pageDescriptor =>
{
    pageDescriptor.Margin(25);

    pageDescriptor.DefaultTextStyle(TextStyle.Default
        .FontSize(6.3f)
        .FontColor(Colors.Grey.Darken2)
        .Medium());

    pageDescriptor
        .Header()
        .ShowOnce()
        .Component(new HeaderComponent(_cfdi));

    pageDescriptor
        .Content()
        .Component(new ContentComponent(_cfdi));
```

And I also had this code snippet at the bottom where the spans are, but now it doesn't work

```
td.Cell().ColumnSpan(13).PaddingTop(10f).PaddingLeft(3f).DefaultTextStyle(TextStyle.Default.FontSize(5).WrapAnywhere()).Column(cd =>
{
    cd.Item().Text(td =>
    {
... more code
```

**Comment by MarcinZiabek** [comment]
I am hoping to release fix today / tomorrow. I am sorry for this inconvenience. But hey, great news! No code changes required, just update the nuget 😁

**Comment by MarcinZiabek** [comment]
This bug should be fixed in the 2022.8.1 release 😁
